src/auro-select.js

```diff
@@ -37,8 +34,10 @@ import styleCss from "./style-css.js";
  * @slot - Default slot for the menu content.
  * @slot label - Defines the content of the label.
  * @slot helpText - Defines the content of the helpText.
- * @fires auroSelect-ready - Notifies that the component has finished initializing.
- * @fires auroSelect-valueSet - Notifies that the component has a new value set.
+ * @event auroSelect-ready - Notifies that the component has finished initializing.
+ * @event auroSelect-valueSet - Notifies that the component has a new value set.
+ * @event auroSelect-validated - Notifies that the `validity` value has changed.
+ * @event auroSelect-helpText - Notifies that the helpText has changed.
```

Comment:
Web Component Analyzer does not support `@event` in its API https://www.npmjs.com/package/web-component-analyzer#-how-to-document-your-components-using-jsdoc

The recommendation is to use `@fires`.